package.json

```diff
@@ -113,7 +115,12 @@
 			{
 				"category": "MongoDB",
 				"command": "cosmosDB.executeMongoCommand",
-				"title": "Execute Command"
+				"title": "Execute Mongo Command"
```

Comment:
The Mongo/MongoDB is inconsistent.  For instance, it's "New Mongo Scrapbook", and the name of the "language" is currently "mongo", and the extension is "*.mongo".  I would have chosen MongoDB, but not sure what it should be now.

Added Mongo to the command because it's not very clear that the context menu in the scrapbook refers to this.  There can be lots of other commands in there, too, and simply "Execute Command" is rather vague (I passed it up the first time I saw it).

![image](https://user-images.githubusercontent.com/6913354/38640959-9ef8fa8e-3d8a-11e8-9638-fd6277afd101.png)